devicetypes/smartthings/fibaro-smoke-sensor.src/fibaro-smoke-sensor.groovy

```diff
@@ -119,9 +119,28 @@ metadata {
 
 def updated() {
 	log.debug "Updated with settings: ${settings}"
+	if(!state.legacySettingsUpdated) updateLegacySettings()
 	setConfigured("false") //wait until the next time device wakeup to send configure command
 }
 
+def updateLegacySettings() {
```

Comment:
here's what I came up with.

other option is just to have this be called from the wakeup handler, then we wouldn't have to manually call update

Reply:
This doesn't look like it's actually changing the current setting value, is it?

Reply:
you are correct. fixed